src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DependencyFlowGraph.cs

```diff
@@ -127,6 +128,36 @@ public void RemoveEdge(DependencyFlowEdge edge)
             }
         }
 
+        public async Task MarkToolingEdges(IRemoteFactory remoteFactory, ILogger logger)
+        {
+            var stopwatch = new Stopwatch();
```

Comment:
Stopwatch?

Reply:
I used it for testing and forgot to remove it.